### PR TITLE
make some updates to snowDepth from ADPSFC

### LINKED ISSUE #1259: initial pathway for snowDepth observation from NCEP BUFR ADPSFC
## Description
create a pathway for snowDepth from NCEP BUFR ADPSFC files

## Requirements

NCEP BUFR ADPSFC data and working bufr2ioda library which uses NCEP BUFRlib

## Acceptance Criteria (Definition of Done)
IODA file created

### PR DESCRIPTION
**Pull request by BenjaminRuston**
## Description

incorporate some updates to the IODA encoding of snowDepth from the NCEP BUFR ADPSFC file

## Issue(s) addressed

Resolves #1259

### CODE REVIEW

**Comment by BenjaminRuston** [comment]
still a work in progress, at the stage where considering updating the reference file

the yaml in this example should be closer to what you are using now @jiaruidong2017 and @YoulongXia-NOAA

**Comment by YoulongXia-NOAA** [comment]
@BenjaminRuston @rmclaren, I am not sure what purpose of this development is. I found air temperature and dew point temperature has been removed from adpsfc yaml file but only keep snow depth. The means that land DA will not use screen-level air temperature and dew point temperature in the future or just do a test for land snow DA.  My original adpsfc snow yaml file development includes these two variables as the users plan to use.

File: test/testinput/bufr_ncep_snow_adpsfc.yaml [hunk 1]
```diff
@@ -23,26 +23,22 @@ observations:
             query: "[*/CLON, */CLONH]"
 
           latitude:
-            query:  "[*/CLAT, */CLATH]"
-
-          stationPressure:
-            query: "*/PRES"
```
Comment (BenjaminRuston) [inline test/testinput/bufr_ncep_snow_adpsfc.yaml:29]: should add pressure back

File: test/testinput/bufr_ncep_snow_adpsfc.yaml [hunk 2]
```diff
@@ -39,10 +39,15 @@ observations:
 
           dewpointTemperature:
             query: "*/TMDP"
-        
+
           totalSnowDepth:
             query: "[*/SNWSQ1/TOSD, */MTRMSC/TOSD, */STGDSNDM/TOSD]"
-
+            transforms:
+              - scale: 1000.0
```
Comment (BenjaminRuston) [inline test/testinput/bufr_ncep_snow_adpsfc.yaml:46]: @YoulongXia-NOAA this file was given to me I thought as an update

however with the changes this corresponds to the unit change to 'mm' reverting this but do like the upperBound will scale it into meters

**Comment by BenjaminRuston** [comment]
@YoulongXia-NOAA agree.  I was provided a yaml for decoding, and this was different than what was currently in the repository

the repository version kept these parameters so seems like it should be kept.  at this point looks like nothing besides some small inconsequential formatting

**Pull request closed without merging**